test/fuzzing/meson.build

```diff
@@ -66,11 +57,61 @@ test('repacker-fuzzer', find_program('run-fuzzer-tests.py'),
   suite: ['fuzzing'],
 )
 
-test('draw-fuzzer', find_program('run-fuzzer-tests.py'),
+test('draw-fuzzer', run_fuzzer_tests,
   args: [
     hb_draw_fuzzer_exe,
     meson.current_source_dir() / 'fonts',
   ],
   workdir: meson.current_build_dir() / '..' / '..',
   suite: ['fuzzing'],
 )
+
+ls = find_program('ls')
+if not ls.found()
+  # On Windows, we need to use the 'dir' command to list files in a directory.
+  # The 'ls' command is not available by default.
+  find = [find_program('dir', required: true), '/b', '/a-d']
+else
+  # On other platforms, we can use the 'ls' command.
+  find = [find_program('ls', required: true)]
+endif
```

Comment:
Not sure if I like this. It means tests will now fail on non-unix non-Windows systems (no idea if these actually exist and matter). What about using Python to glob the dir? A small Python program that calls `pathlib.Path().iterdir()`?

Reply:
Or since meson really [doesn’t want us to glob paths](https://mesonbuild.com/FAQ.html#why-cant-i-specify-target-files-with-a-wildcard), may be it is time to bite the bullet and list all the fuzzing fonts in the meson file.

Reply:
If we do list all fonts, we can also have a test that make sure they are all in git...